src/configuration.ts

```diff
@@ -34,13 +34,16 @@ export const DEFAULT_CONFIG = {
         "quotemark": [true, "double"],
         "semicolon": [true, "always"],
         "triple-equals": [true, "allow-null-check"],
-        "typedef-whitespace": [true, {
-            "call-signature": "nospace",
-            "index-signature": "nospace",
-            "parameter": "nospace",
-            "property-declaration": "nospace",
-            "variable-declaration": "nospace",
-        }, ],
+        "typedef-whitespace": [
+            true,
+            {
+                "call-signature": "nospace",
+                "index-signature": "nospace",
+                "parameter": "nospace",
+                "property-declaration": "nospace",
+                "variable-declaration": "nospace",
+            },
+        ],
```

Comment:
This alignment is changed to avoid vscode constantly kill the space in the original line 43: `},<space>],` when execute `format code`, and causes build error.